tests/test_complexes.py

```
@@ -0,0 +1,678 @@
+                ),
+            )
+
+        # in this case, two of the molecules are actually equivalent, with slightly different ES energies
+        # because of the periodicity breaking in the generated supercell:
+        for key, mol in molecule_dict.items():
+            if key in [-4245.99, -4246.28]:
+                assert are_equivalent_molecules(molecule_dict[-4245.99], mol)
+            else:
+                assert not are_equivalent_molecules(molecule_dict[-4245.99], mol)
+
+    def test_symmetry_multiplicity_C2m_Ga2O3(self):
```

Pylint (3.3.7)

[refactor] Too many local variables (28/15)

(R0914)